src/server/controller/cycleData/persistNodeValue/calculateDependantNodes/calculateDependantNodes.ts

```diff
@@ -7,7 +7,10 @@ import { RowRepository } from '@server/repository/assessment/row'
 
 import { calculateNode } from './calculateNode'
 
-export const calculateDependantNodes = async (props: Props, client: BaseProtocol): Promise<NodeUpdates> => {
+export const calculateDependantNodes = async (
+  props: Omit<Props, 'value'>,
```

Comment:
check if value is not needed in src/server/controller/cycleData/persistNodeValue/index.ts

Reply:
It is not used
https://github.com/openforis/fra-platform/blob/e2031fc11b7e832653914f6545f59a1081fce7ff/src/server/controller/cycleData/persistNodeValue/index.ts#L18